src/main/java/frc/robot/subsystems/instances/Lift.java

```diff
@@ -7,24 +7,119 @@
 
 package frc.robot.subsystems.instances;
 
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.Solenoid;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.subsystems.interfaces.LiftInterface;
 
 public class Lift extends SubsystemBase implements LiftInterface {
   /**
    * Creates a new Lift.
    */
+  private static WPI_TalonSRX leftMotorLeader;
+  private static WPI_TalonSRX rightMotorLeader;
+  private static WPI_TalonSRX leftMotorFollower;
+  private static WPI_TalonSRX rightMotorFollower;
+  private static Compressor compressor;
+  private static Solenoid solenoid;
+
+
   public Lift() {
+    leftMotorLeader = new WPI_TalonSRX(12);
+    rightMotorLeader = new WPI_TalonSRX(13);
+    leftMotorFollower = new WPI_TalonSRX(14);
+    rightMotorFollower = new WPI_TalonSRX(15);
+
+    leftMotorLeader.configFactoryDefault();
+    rightMotorLeader.configFactoryDefault();
+    leftMotorFollower.configFactoryDefault();
+    rightMotorFollower.configFactoryDefault();
+
+    leftMotorLeader.setSafetyEnabled(true);
+    rightMotorLeader.setSafetyEnabled(true);
+    leftMotorFollower.setSafetyEnabled(true);
+    rightMotorFollower.setSafetyEnabled(true);
+
+    leftMotorLeader.setNeutralMode(NeutralMode.Brake);
+    rightMotorLeader.setNeutralMode(NeutralMode.Brake);
+    leftMotorFollower.setNeutralMode(NeutralMode.Brake);
+    rightMotorFollower.setNeutralMode(NeutralMode.Brake);
+
+    leftMotorLeader.configPeakOutputForward(1.0);
+    rightMotorLeader.configPeakOutputReverse(-1.0);
+    leftMotorFollower.configPeakOutputForward(1.0);
+    rightMotorFollower.configPeakOutputReverse(-1.0);
+
+    //leftMotorLeader.setInverted(true);
```

Comment:
Whadisdis

Reply:
idc

Reply:
Pleas4 remove

Reply:
We weren't sure if we were going to need it so we commented it out just in case.